XiEditor/EditViewController.swift

```diff
@@ -32,9 +32,20 @@ class EditViewController: NSViewController, EditViewDataSource {
     @IBOutlet weak var editContainerView: EditContainerView!
     @IBOutlet var editView: EditView!
     @IBOutlet weak var gutterView: GutterView!
+    @IBOutlet weak var stackView: NSStackView!
```

Comment:
I'm raising an eyebrow at this choice, seems like not a great fit for this problem? But 🤷‍♂️.